index.js

```diff
@@ -1,18 +1,130 @@
-const MangoTree = require('./mango_tree.js');
-const AppleTree = require('./apple_tree.js');
-const PearTree = require('./pear_tree.js');
+const MangoTree = require('./trees/mango-tree');
+const AppleTree = require('./trees/apple-tree');
+const PearTree = require('./trees/pear-tree');
 
-class TreeGrove {}
+class TreeGrove {
+    constructor(){
+        this._allTrees = []
+        // this._age = age
+        // this._height = height
+        // this._matureAge = matureAge
+        // this._healthStatus = healthStatus
+    }
+
+    get allTrees (){
+        return this._allTrees
+    }
+    // get age (){
+    //     return this._age
+    // }
+    // get height (){
+    //     return this._height
+    // }
+    // get matureAge (){
+    //     return this._matureAge
+    // }
+    // get healthStatus (){
+    //     return this._healthStatus
+    // }
+
+    set allTrees (val){
+        this._allTrees = val
+    }
+    // set age (val){
+    //     this._age = val
+    // }
+    // set height (val){
+    //     this._height = val
+    // }
+    // set matureAge (val){
+    //     this._matureAge = val
+    // }
+    // get healthStatus (val){
+    //     this._healthStatus = val
+    // }
+
+    inputTree(name, age, height, matureAge, healthStatus){
+        if(name === "MangoTree"){
+            let temp = []
+            temp.push(name)
+            temp.push(new MangoTree(age, height, healthStatus, matureAge))
+            this.allTrees.push(temp)
```

Comment:
harusnya this._alltress itu nyimpen arr of instance treesnya ... ga usah arr 2 dimensi